.travis.yml

```diff
@@ -1,35 +1,29 @@
 language: android
```

Comment:
This file is auto-generated by Travis tooling when generating the encrypted key, so hence the "unnecessary" formatting diffs.